packages/server/postgres/migrations/1674237759799_meetingTemplates.ts

```diff
@@ -0,0 +1,47 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DROP TYPE IF EXISTS "SharingScopeEnum";
+    CREATE TYPE "SharingScopeEnum" AS ENUM (
+      'USER',
```

Comment:
I added User to the ReBAC model here even though we don't use it yet. The goal is to use the same `SharingScopeEnum ` across many different entities in our system